crowdsource/issues.py

```diff
@@ -0,0 +1,9 @@
+from dataquality.issues import IssueType
+
+# Issues with related_class "community-person"
+
+IssueType('wrong-phone', 'Wrong Phone Number', 'community-person', 'error')
```

Comment:
we should document why community-person is used instead of person

Reply:
Sure.